pkg/serverless/apikey/env_test.go

```diff
@@ -31,6 +40,38 @@ func TestGetSecretEnvVars(t *testing.T) {
 	assert.Equal(t, map[string]string{
 		"TEST_KMS":   "DECRYPTED_VAL",
 		"TEST_SM":    "DECRYPTED_VAL",
-		"DD_API_KEY": "DECRYPTED_VAL",
+		apiKeyEnvVar: "DECRYPTED_VAL",
 	}, decryptedEnvVars)
 }
+
+func TestDDApiKey(t *testing.T) {
+	t.Setenv(apiKeyEnvVar, "abc")
+	assert.NoError(t, HandleEnv())
+}
+
+func TestHasDDApiKeySecretArn(t *testing.T) {
+	t.Setenv(apiKeySecretManagerEnvVar, "abc")
+	mockSetSecretsFromEnv(t, os.Environ())
+	assert.NoError(t, HandleEnv())
```

Comment:
This might look a little awkward, but I wanted to test `HandleEnv()` and had to handle the KMS + Secret Manager decryption functions